### PR TITLE
Add option to ignore @internal classes and methods

### PR DESCRIPTION
**Pull request by tdely**
Adds the option `--no-internal` to ignore classes and/or methods marked `@internal`.

### CODE REVIEW

File: src/PHPDocsMD/ClassEntity.php [hunk 1]
```diff
@@ -69,6 +74,19 @@ public function hasIgnoreTag($toggle=null)
...
 
+    /**
+     * @param bool $toggle
+     * @return bool
+     */
+    public function hasInternalTag($toggle = null)
+    {
+        if ($toggle === null) {
+            return $this->hasInternalTag;
+        } else {
```
Comment (bueltge) [inline src/PHPDocsMD/ClassEntity.php:85]: The `else` is not necessary. After the if statement the return is sufficiently. Makes easier code, more maintainable and testable.  The second return works if the the If-statement is not true.

File: src/PHPDocsMD/CodeEntity.php [hunk 2]
```diff
@@ -47,6 +52,19 @@ public function isDeprecated($toggle=null)
...
 
+    /**
+     * @param bool $toggle
+     * @return bool|null
+     */
+    public function isInternal($toggle=null)
+    {
+        if ($toggle === null) {
+            return $this->isInternal;
+        } else {
```
Comment (bueltge) [inline src/PHPDocsMD/CodeEntity.php:63]: Also here, the follow example is sufficiently.
```php
        if ($toggle === null) {
            return $this->isInternal;
        }

        return $this->isInternal = (bool)$toggle;

```

File: src/PHPDocsMD/ClassEntity.php (hunk 1 above)
Comment (tdely) [reply src/PHPDocsMD/ClassEntity.php:85]: This was done only to match the rest of the file.

File: src/PHPDocsMD/CodeEntity.php (hunk 2 above)
Comment (tdely) [reply src/PHPDocsMD/CodeEntity.php:63]: Again, this was done only to match the rest of the file.

**Pull request merged**